### PR TITLE
Added optional reason for ?ban command

### PR DESCRIPTION
**Pull request by ondra05**
- Added optional reason for ?ban command
- Added extra space in front of hammer emoji like Python version does

### CODE REVIEW

File: src/moderation.rs [hunk 1]
```diff
@@ -109,23 +109,28 @@ pub fn joke_ban(args: &Args) -> Result<(), Error> {
...
         Some(bannee) => crate::api::send_reply(
             args,
             &format!(
-                "{}#{} banned user {}#{} {}",
+                "{}#{} banned user {}#{}{}  {}",
                 args.msg.author.name,
                 args.msg.author.discriminator,
                 bannee.name,
                 bannee.discriminator,
+                reason.unwrap_or("".to_string()),
```
Comment (coolshaurya) [inline src/moderation.rs:133]: you can do `.unwrap_or_default()` too

Comment (coolshaurya) [reply src/moderation.rs:133]: oh lol it has been merged

Comment (kangalioo) [reply src/moderation.rs:133]: Yes it's already merged heh

Sorry, I didn't know people were about to review the PR

Comment (coolshaurya) [reply src/moderation.rs:133]: No need to be sorry 💛. I don't think a single comment qualifies as a "review" 😄.

**Pull request merged**